docs/BreakingChanges/142 WCF Binding with the TransportWithMessageCredential security mode.md

```diff
@@ -30,7 +30,6 @@ Because this is an opt-in feature, it should not affect the behavior of existing
 ### Affected APIs
 * `F:System.ServiceModel.BasicHttpSecurityMode.TransportWithMessageCredential`
 * `F:System.ServiceModel.BasicHttpsSecurityMode.TransportWithMessageCredential`
-* `F:System.ServiceModel.PollingDuplexHttpSecurityMode.TransportWithMessageCredential`
```

Comment:
This is only in Silverlight, so it won't ever be found in .NET framework desktop apps